### DISCUSSION TITLE
MacPreview - How to enable autoplay for video and audio?

### DISCUSSION THREAD

**Discussion by VicDobrov** [doublecmd/doublecmd#1451] [General]
The disadvantage of MacPreview.wlx is that after pressing F3 (Ctrl+Q) you then need to press the "Play" button.

**How to enable autoplay in MacPreview?**
my System: MacOS 13.6, MacMini M2 Pro

**Comment by rich2014** [comment]
@VicDobrov 

done.

download from https://doublecmd.sourceforge.io/site/eng/snapshots.php and test it.

**Comment by VicDobrov** [comment]
**rich2014** Thank you!
Now music and videos are autoplay!

**Comment by rich2014** [comment]
it's great.
